### ISSUE TITLE
where is the "dual loss function" ?

### ISSUE THREAD

**Issue by boogieLing** [kaist-dmlab/DF-TAR#8]
hallo, in your paper, you mentioned the use of "dual loss function" as the loss of the training process.
Where is this implementation in the code. thank you.
